File: Python/Pedido.py
class Pedido:
    """Representa un pedido con un ID y el nombre del plato."""

    def __init__(self, id, nombre_plato):
         
         
        """ Inicializa un pedido.
        :param id: Identificador numérico del pedido.
        :param nombre_plato: Nombre del plato solicitado.
        """
        self.id = id
        self.nombre_plato = nombre_plato
        
        """Devuelve una representación legible del pedido."""
        
    def __str__(self):
        return f"Pedido #{self.id} ({self.nombre_plato})"

File: Pedido.java
/**
 * Representa un pedido con un id y el nombre del plato.
 */
public class Pedido {
    private final int id;// Identificador único del pedido
    private final String nombrePlato;// Nombre del plato solicitado

    /**
     * Crea un nuevo pedido.
     * @param id identificador del pedido
     * @param nombrePlato nombre del plato
     */
    public Pedido(int id, String nombrePlato) {
        this.id = id;
        this.nombrePlato = nombrePlato;
    }

    /** @return id del pedido */
    public int getId() {
        return id;
    }

     /** @return nombre del plato */
    public String getNombrePlato() {
        return nombrePlato;
    }

    /** @return representación rápida del pedido */
    @Override
    public String toString() {
        return "Pedido #" + id + " (" + nombrePlato + ")";
    }
}